rwengine/src/render/OpenGLRenderer.hpp

```diff
@@ -425,7 +425,9 @@ class OpenGLRenderer final : public Renderer {
     // Debug group profiling timers
     ProfileInfo profileInfo[MAX_DEBUG_DEPTH];
     GLuint debugQuery;
+#ifdef RW_PROFILER
```

Comment:
This should be under `RW_GRAPHICS_STATS` now.

Reply:
`RW_GRAPHICS_STATS` is defined nowhere.